python/datasources/cdc_hiv.py

```diff
@@ -10,30 +10,45 @@
 from ingestion import gcs_to_bq_util, standardized_columns as std_col
 from ingestion.merge_utils import merge_county_names
 
-
+# constants
 HIV_DIR = 'cdc_hiv'
+DIAGNOSES = 'diagnoses'
+DEATHS = 'deaths'
+PREP = 'prep'
+DTYPE = {'FIPS': str, 'Year': str}
+NA_VALUES = ['Data suppressed', 'Data not available']
 
 # a nested dictionary that contains values swaps per column name
-HIV_TERMS_STANDARD_BY_COL = {
+BREAKDOWN_TO_STANDARD_BY_COL = {
     std_col.AGE_COL: {'Ages 13 years and older': std_col.ALL_VALUE},
-    std_col.HIV_DIAGNOSES: {'Data suppressed': np.nan, 'Data not available': np.nan},
-    std_col.HIV_DIAGNOSES_PER_100K: {'Data suppressed': np.nan, 'Data not available': np.nan},
     std_col.RACE_CATEGORY_ID_COL: {
-        'All races/ethnicities': std_col.Race.ALL,
+        'All races/ethnicities': std_col.Race.ALL.value,
         'American Indian/Alaska Native': std_col.Race.AIAN_NH.value,
         'Asian': std_col.Race.ASIAN_NH.value,
         'Black/African American': std_col.Race.BLACK_NH.value,
         'Hispanic/Latino': std_col.Race.HISP.value,
-        'Multiracial': std_col.Race.MULTI_NH.value,
+        'Multiracial': std_col.Race.MULTI_OR_OTHER_STANDARD_NH,
```

Comment:
I noticed in your comment that this is likely the best categorization for all three topics. It's OK for the topics to have different categorizations if that is correct, lets just make sure this is right.